### PR TITLE
CI Link checker, Update with changes from Dev

### PR DESCRIPTION
**Pull request by budak7273**

### CODE REVIEW

File: modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/machines/SolarPanel.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+This value defaults to the value we set in our building,
+but setting it to zero here will prevent the undesired functionality
+while still correctly reporting the power production in the build gun.
+Hook up the execution pin to Begin Play, since we only need this to happen once - on actor spawn.
+
+image:BeginnersGuide/simpleMod/machines/SolarPanel/DisableDynamicProduction.png[Setting the Dynamic Production]
+
+Next, we need to figure out whether it's day or night and adjust power production accordingly.
+Thankfully, the `FGTimeOfDaySubsystem` offers some helpful methods for that.
+Get an instance of the `FGTimeOfDaySubsystem` with the `GetTimeSubsystem` node to make use of its `Is Day` method.
```
Comment (Feyko) [inline modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/machines/SolarPanel.adoc:63]: `Is Day` could be mentioned later, it confused me to not see it in the subsequent screenshots

File: modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/machines/SolarPanel.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+More info xref:SMLChatCommands.adoc[here].
+=====
+
+If you're having trouble getting your generator to work, check out the completed one in Example Mod,
+or ask for help on the Discord.
+
+== Next Steps
+
+You may wish to add functionality to make the solar panel require direct access to the sky to function,
```
Comment (Feyko) [inline modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/machines/SolarPanel.adoc:114]: I'm guessing the naive implementation would be checking with a trace each tick. Is that so bad for performance?
A more proper implementation could be checking at startup and then only disabling when a buildable is built on top of it. Could be a good exercise to teach about "caching" as well as the buildable built event!

File: modules/ROOT/pages/Development/Satisfactory/Multiplayer.adoc [hunk 3]
```diff
@@ -167,6 +162,7 @@ The ExampleMod uses a replicated property on the Multiplayer Demo Building.
 === {cpp} Example
 
 No example is currently provided.
+Consider looking at an open-source mod instead.
```
Comment (Feyko) [inline modules/ROOT/pages/Development/Satisfactory/Multiplayer.adoc:165]: Link OpenSourceExamples

File: modules/ROOT/pages/Development/UnrealEngine/UObject.adoc
Comment (Feyko) [inline modules/ROOT/pages/Development/UnrealEngine/UObject.adoc:1]: What happened to this?

**Review by Feyko — DISMISSED** [review]
Awesome as always, thanks! That solar panel generator tutorial is quite nice

File: modules/ROOT/pages/Development/UnrealEngine/UObject.adoc
Comment (budak7273) [reply modules/ROOT/pages/Development/UnrealEngine/UObject.adoc:1]: The page didn't offer anything of value; check out the deletion diff

**Review by mircearoata — APPROVED** [review]

**Pull request merged**